### PR TITLE
Overhaul of the desired accounts script.

### PR DESCRIPTION
**Pull request by illfelder**
The change should improve performance by reducing the number of hanging
requests for metadata.

### CODE REVIEW

File: google-daemon/usr/share/google/google_daemon/accounts_manager.py [hunk 1]
```diff
@@ -80,9 +79,7 @@ def Main(self):
           time.sleep(5)
 
         # Write the etag to pass to parent
-        json_tags = json.dumps(
-            [self.desired_accounts.attributes_etag,
-             self.desired_accounts.instance_sshkeys_etag])
+        json_tags = json.dumps([self.desired_accounts.etag])
```
Comment (EricSL) [inline google-daemon/usr/share/google/google_daemon/accounts_manager.py]: Any reason to still send this as a list?

Comment (illfelder) [inline google-daemon/usr/share/google/google_daemon/accounts_manager.py]: Possibly - if a VM upgrades to this change and that file already exists on the machine, I don't want the script to break.

Comment (EricSL) [inline google-daemon/usr/share/google/google_daemon/accounts_manager.py]: It doesn't look to me like it is ever saved to a file -- it is sent over a pipe from the forked child to the parent, so the two ends of the pipe should always be running the same binary.

I guess it's fine either way. LGTM.

Comment (illfelder) [inline google-daemon/usr/share/google/google_daemon/accounts_manager.py]: Mind taking a look at the new version of this file?

**Comment by zmarano** [comment]
LGTM

**Pull request merged**